bigquery/docs/usage/queries.rst

```diff
@@ -30,7 +30,7 @@ Writing query results to a destination table
 See BigQuery documentation for more information on
 `writing query results <https://cloud.google.com/bigquery/docs/writing-results>`_.
 
-.. literalinclude:: ../snippets.py
+.. literalinclude:: ../samples/client_query_destination_table.py
```

Comment:
Regarding the complexity of the PR review, I'm asking to let me not modify the `*.rst` files now to evade cluttering. I will open a new issue and make a PR to cover it after finishing the deprecation of the `snippets.py` file.